spec/features/post_show_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+RSpec.describe 'Post index page', type: :feature do
+  before(:example) do
+    @user1 = User.create!(name: 'Elon Musk',
+                          photo: 'https://pbs.twimg.com/media/FdTUjpUWQAEzO0n.jpg:large',
+                          bio: 'Tweeter.', posts_counter: 56)
+    @post = Post.create!(author: @user1, title: 'Web Development', text: 'Learn Web developmet at Microverse',
+                         comments_counter: 1, likes_counter: 0)
+    Comment.create!(author_id: @user1.id, post_id: @post.id, text: 'this is first comment')
+    visit user_posts_path(user_id: @user1.id, id: @post.id)
+  end
+  describe 'User index page' do
+    it 'I can see the user username.' do
+      expect(page).to have_content(@user1.name)
+    end
+
```

Comment:
- [ ] Great job so far. you are almost there.  for featured tests for User post show page, you are missing test for;
- Tests that asserts you can see who wrote the post.
- Tests that asserts you can see the username of each commentor
- Tests that asserts you can see the comment each commentor left.
please kindly add that.
nice work 👏🏾